config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/dishes/:dish_id", to: "dishes#show"
+  post "/dishes/:dish_id", to: "dish_ingredients#create"
```

Comment:
This path suggests you are creating a dish itself, so a more appropriate path would be "/dishes/:dish_id/ingredients" - but I agree with how you've set up your controller action for it!